libraries/shared/storage/backfill/storage_value_loader.go

```diff
@@ -53,80 +64,96 @@ func (r *StorageValueLoader) Run() error {
 	}
 
 	for _, header := range headers {
-		for address, chunkedKeys := range addressToKeys {
-			for _, keys := range chunkedKeys {
-				persistStorageErr := r.getAndPersistStorageValues(address, keys, header.BlockNumber, header.Hash)
-				if persistStorageErr != nil {
-					return persistStorageErr
-				}
-			}
+		persistStorageErr := r.getAndPersistStorageValues(header.BlockNumber, header.Hash)
+		if persistStorageErr != nil {
+			return persistStorageErr
 		}
+
 	}
-	logrus.Infof("Finished persisting storage values for %v addresses from block %v to %v.", len(addressToKeys), r.startingBlock, r.endingBlock)
+	logrus.Infof("Finished persisting storage values for %v addresses from block %v to %v.", len(r.addressesToKeysToValues), r.startingBlock, r.endingBlock)
 
 	return nil
 }
 
-func (r *StorageValueLoader) getStorageKeys() (map[common.Address][][]common.Hash, error) {
-	addressToKeys := make(map[common.Address][][]common.Hash, len(r.initializers))
+func (r *StorageValueLoader) setAddressesToKeysToValues() error {
```

Comment:
The way I'm reading this method is that it is adding the keys to `addressesToKeysToValues` with empty values, is that correct? I wonder if calling this method `addKeysToAddressesToKeysToValues` would make sense as a way to specify that the keys are being added but the values are empty? I think upon first looking at it, I thought that the key->values would be set. 

But then again `addKeysToAddressesToKeysToValues` is a lot - I wonder if there's a name we can use for the concept of `addressesToKeysToValues` to simplify things? 🤔 Maybe `storageByAddress`? And then this method could be `addKeysToStorageByAddress`? Not sure if that's any better though, so feel free to disregard this rambling. 😜

Reply:
Really dig the suggested renaming 👍